pydev_ipython/inputhookmac.py

```diff
@@ -0,0 +1,167 @@
+"""Inputhook for OS X
+
+Calls NSApp / CoreFoundation APIs via ctypes.
+"""
+
+import os
+from pydev_ipython.inputhook import stdin_ready
+import time
+from threading import Thread, Event
```

Comment:
Can you change this import for: `from _pydev_imps._pydev_saved_modules.threading import Thread, Event` ?

The main issue is that gevent monkey-patches threading and that import will make sure that the original (non monkey-patched version) of the module is used.

Reply:
Please see the commit 7f755ca [1].

[1] https://github.com/fabioz/PyDev.Debugger/pull/156/commits/7f755cabccea3de3461ff2c0beec1d924a85e5a4